yt_dlp/extractor/abematv.py

```diff
@@ -284,10 +223,82 @@ def _get_media_token(self, invalidate=False, to_show=True):
                 'appId': 'tv.abema',
                 'appVersion': '3.27.1'
             }, headers={
-                'Authorization': 'bearer ' + self._get_device_token()
+                'Authorization': f'bearer {self._get_device_token()}',
             })['token']
 
-        return self._MEDIATOKEN
+        return AbemaTVBaseIE._MEDIATOKEN
+
+    def _call_api(self, endpoint, video_id, query=None, note='Downloading JSON metadata'):
+        return self._download_json(
+            f'https://api.abema.io/{endpoint}', video_id, query=query or {},
+            note=note,
+            headers={
+                'Authorization': f'bearer {self._get_device_token()}',
+            })
+
+    def _extract_breadcrumb_list(self, webpage, video_id):
+        for jld in re.finditer(
+                r'(?is)</span></li></ul><script[^>]+type=(["\']?)application/ld\+json\1[^>]*>(?P<json_ld>.+?)</script>',
+                webpage):
+            jsonld = self._parse_json(jld.group('json_ld'), video_id, fatal=False)
+            if jsonld:
+                if jsonld.get('@type') != 'BreadcrumbList':
+                    continue
+                trav = traverse_obj(jsonld, ('itemListElement', ..., 'name'))
+                if trav:
+                    return trav
```

Comment:
```suggestion
            if traverse_obj(jsonld, '@type') != 'BreadcrumbList':
                continue
            trav = traverse_obj(jsonld, ('itemListElement', ..., 'name'))
            if trav:
                return trav
```
what does `trav` mean?

Reply:
**trav**erse